scripts/sync_fofb/scan_dcc_links.py

```diff
@@ -0,0 +1,101 @@
+#! /usr/bin/python3
+
+''' Script to scan FOFB DCC FMC/P2P links.
+
+    This script reads DCC link partners PVs, builts a directed graph and saves
+    it on disk using json format.
+
+    Usage:  "./scan_dcc_links.py <JSON FILENAME> <LIST OF CRATES>", where
+
+            <JSON FILENAME>:    output filename
+            <LIST OF CRATES>:   list of crates' indexes separated by spaces
+
+    Author: Guilherme Ricioli
+'''
+
+import collections
+from epics import PV
+from itertools import product
+import json
+import sys
+
+import bpm_epics_ioc_slot_mapping
+
+# crates
+crates = [f"{int(i):02}" for i in sys.argv[2:]]
+
+# logical slots
+rtmlamp_slot = 3
+slots = [rtmlamp_slot, 13, 15, 17, 19]
```

Comment:
This has the same issue as the sync script, which is not taking into account slot 6 (number 11). It might make sense for us to start thinking about a solution, unless that the ID BPMs don't matter here?

Reply:
Indeed, they should be taken into account. We can further discuss this in person.